.github/workflows/build_and_test.yml

```diff
@@ -367,7 +367,9 @@ jobs:
           - >-
             pyspark-pandas-connect
```

Comment:
totally nit: to be consistent, what about:
```
pyspark-pandas-connect-part0
pyspark-pandas-connect-part1
pyspark-pandas-connect-part2
```
?

Reply:
Renamed. Thanks!